CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 1.1.1 (Unreleased)
 
+BUG FIXES
+* Fixes no more 'force new resource' for container ports when
+there are no changes. This was caused to the ascending order. See [GH-110] 
+for details and [[#115](https://github.com/terraform-providers/terraform-provider-docker/pull/115)]
+
```

Comment:
Nitpick: We usually update the changelog _after_ a PR is merged as it helps everyone avoiding merge conflicts.

Reply:
Thanks for the hint, I'll keep this in mind